samples/cellular/nrf_cloud_rest_cell_location/README.rst

```diff
@@ -25,60 +20,43 @@ The sample supports the following development kits:
+If it is not, follow the instructions in `Device on-boarding <nrf_cloud_rest_cell_location_onboarding>`_.
 
 Overview
 ********
 
-After the sample initializes and connects to the network, it enters single-cell mode and sends a single-cell location request to nRF Cloud using network data obtained from the :ref:`modem_info_readme` library.
-To enable multi-cell mode, press **Button 1**.
-In multi-cell mode, the sample requests for neighbor cell measurement using the :ref:`lte_lc_readme` library.
-If neighbor cell data is measured, the sample sends a multi-cell location request to nRF Cloud.
-Otherwise, the request is single-cell.
-In either mode, the sample sends a new location request if a change in cell ID is detected.
+The sample can work in two modes: single-cell mode and multi-cell mode.
 
-See the `nRF Cloud Location Services documentation`_ for additional information.
-
-Limitations
-***********
+After the sample initializes and connects to the network, it enters the single-cell mode and sends a single-cell location request to nRF Cloud.
+For this purpose, the sample uses network data obtained from the :ref:`modem_info_readme` library.
+In the multi-cell mode, the sample requests for neighbor cell measurement using the :ref:`lte_lc_readme` library.
 
-The sample requires the network carrier to provide date and time to the modem.
-Without a valid date and time, the modem cannot generate JWTs with an expiration time.
-
-User interface
-**************
+If the modem provides neighbor cell data, the sample sends a multi-cell location request to nRF Cloud.
+Otherwise, the request is single-cell.
 
-Pressing **Button 1** toggles between single-cell and multi-cell mode.
+In either mode, the sample sends a new location request if a change in cell ID is detected.
 
-Set the :ref:`CONFIG_REST_CELL_CHANGE_CONFIG <CONFIG_REST_CELL_CHANGE_CONFIG>` Kconfig config value to try all combinations of the :c:struct:`nrf_cloud_location_config` structure values ``hi_conf`` and ``fallback``.
+See the `nRF Cloud Location Services documentation`_ for additional information.
 
-.. _nrf_cloud_rest_location_onboarding:
+.. _nrf_cloud_rest_cell_location_onboarding:
 
-Onboarding your device to nRF Cloud
-***********************************
+Setup
+=====
 
 You must onboard your device to nRF Cloud for this sample to function.
 You only need to do this once for each device.
 To onboard your device, install `nRF Cloud Utils`_ and follow the instructions in the README.
 
 Configuration
 *************
-|config|
 
+|config|
 
 Configuration options
 =====================
 
 Check and configure the following Kconfig options for the sample:
 
-.. _CONFIG_REST_CELL_CHANGE_CONFIG:
-
-CONFIG_REST_CELL_CHANGE_CONFIG - Enable changing request configuration
-   Set this to use the next combination of ``hi_conf`` and ``fallback`` flags after performing both single- and multi-cell location requests.
-
 .. _CONFIG_REST_CELL_DEFAULT_DOREPLY_VAL:
```

Comment:
Also here, you can use the  .. options-from-kconfig instead of listing the options below.